addons/hr_timesheet_attendance/tests/test_timesheet_attendance.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+from datetime import datetime
+
+from odoo.addons.hr_timesheet.tests.test_timesheet import TestCommonTimesheet
+
+class TestTimesheetAttendance(TestCommonTimesheet):
+
+    @classmethod
```

Comment:
You remove this blank space. 🙂 
```suggestion
    @classmethod
```